src/Calendar.coffee

```diff
@@ -97,7 +97,9 @@ class CalendarExtension extends Calendar
 		if resource
 			eventResourceField = @resourceManager.getEventResourceField()
 			@clientEvents (event) -> # return value
-				event[eventResourceField] == resource.id
+				if eventResourceField == 'resourceId' && !event[eventResourceField]
+					eventResourceField='resourceIds';
```

Comment:
resetting `eventResourceField` here will have strange side effects. it is meant to be the value defined by the [eventResourceField](http://fullcalendar.io/docs/resource_data/eventResourceField/) option and i don't see why it should mutate on each iteration of an event.

I've made the precedent here that [resourceId](https://github.com/fullcalendar/fullcalendar-scheduler/blob/master/src/ResourceGrid.coffee#L25-L32) should take precedence over `resourceIds` so can we make the if statement reflect that?

```coffee
@clientEvents (event) -> # return value
    if event[eventResourceField]
        event[eventResourceField] == resource.id
    else if $.isArray(event.resourceIds)
        $.inArray(resource.id, event.resourceIds)
    else
        false
```